server/src/lib/model/achievements.ts

```diff
@@ -73,9 +74,9 @@ const bonus_condition_sql = {
     GROUP BY user_clients.client_id
     `,
   // Arguments: [client_id, client_id, challenge]
-  // Check if user has already earned the bonus
+  // Check if user has already earned the bonus, and if challenge is on-going now
   invite_send: `
-    SELECT earn.id IS NULL AS win_bonus, ? AS bonus_winner
+    SELECT (earn.id IS NULL) AND (NOW() BETWEEN start_date AND TIMESTAMPADD(WEEK, 3, start_date)) AS win_bonus, ? AS bonus_winner
```

Comment:
This should still be achievable before the challenge. We have a pre-challenge state that includes the invite modal.

Reply:
Good point.